apps/signing/views.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import status
+from rest_framework.response import Response
+
+from api.jwt_auth.views import JWTProtectedView
+from addons.models import Addon
+from devhub.views import handle_upload
+
+
+class UploadAddonView(JWTProtectedView):
```

Comment:
Is that meant to be a "generic" name? It could be:
1/ uploading a new add-on (which creates a new version, and at least one new file)
2/ uploading a new version for an add-on (which creates a new version, and at least one new file)
3/ uploading a new file for a version (which creates at least one new file)

Is this view/endpoint going to deal with all three cases at the end of the day (I see it's not at the moment, it's only dealing with 2/, but maybe it will using a POST)? If not, I'd propose the following naming:
1/ UploadAddon
2/ UploadVersion
3/ UploadFiles

As a side note, I said "at least one new file" because in the devhub if you upload a new add-on/version/file, you can select multiple platforms, and one file will be created of each.